StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetFlowControllerViewController.swift

```diff
@@ -343,7 +343,9 @@ class PaymentSheetFlowControllerViewController: UIViewController, FlowController
 
         switch mode {
         case .selectingSaved:
-            headerLabel.text = .Localized.select_your_payment_method
+            headerLabel.text = STPLocalizedString(
+                "Select your payment method",
+                "Title shown above a carousel containing the customer's payment methods")
```

Comment:
Wait this felt like a good change, can we keep it?

Reply:
Sure!